src/pytypes/onefuzztypes/models.py

```diff
@@ -799,6 +799,11 @@ class NetworkConfig(BaseModel):
     subnet: str = Field(default="10.0.0.0/16")
 
 
+class NetworkSecurityGroupConfig(BaseModel):
+    allow_service_tags: Optional[List[str]]
+    allow_IPs: List[str] = Field(default=[])
```

Comment:
No _we do_ want the list to be empty. An empty list means that no src IPs are allowed through the NSG.

Reply:
Doesn't that leave the service by default as "nobody can connect to the service"?

Reply:
"Nobody can connect to debug proxy", which is the behavior we agreed on in our previous conversation. We are likely making the 'set_nsg_rules' param in deploy.py required so that admins know that they'd be locking down by default.